cdap-security/src/test/java/co/cask/cdap/security/authorization/AuthorizerSupplierTest.java

```diff
@@ -59,9 +59,9 @@
 import javax.annotation.Nullable;
 
 /**
- * Tests for {@link AuthorizerInstantiatorService}.
+ * Tests for {@link AuthorizerSupplier}.
  */
-public class AuthorizerInstantiatorServiceTest {
+public class AuthorizerSupplierTest {
```

Comment:
would it be better to just test AuthorizerInstantiatorService in these tests? Is that easy? Primarily because this test only tests the functionality of the AuthorizerInstantiatorService, and not AuthorizerSupplier, right?

We could write more tests for AuthorizerSupplier later, in a separate AuthorizerSupplierTest?